app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= render 'shared/partials/nav-global.html' %>
+<main class="devices">
+  <%= render 'shared/partials/header-global.html' %>
+  <nav class="device__selection">
+    <ul>
+      <li class="device__keurig device-selected">
+        <a href="/device-keurig">
+          <%= image_tag("illustrations/ill-keurig.svg", :alt => "Keurig") %>
```

Comment:
No need to use Ruby here. It's cleaner to do `<img src = "illustrations/ill-keurig.svg" alt = "">` since the asset pipeline will take care of things.